integration/raft/channel_participation_test.go

```diff
@@ -254,7 +254,114 @@ var _ = Describe("ChannelParticipation", func() {
+
+				By("checking the channel height")
+				channelInfo := channelparticipation.ListOne(network, o, "participation-trophy")
+				Expect(channelInfo).To(Equal(expectedChannelInfoPT))
+			}
+
+			By("submitting a channel config update")
+			channelConfig := nwo.GetConfig(network, peer, orderer1, "participation-trophy")
+			c := configtx.New(channelConfig)
+			err := c.Orderer().AddCapability("V1_1")
+			Expect(err).NotTo(HaveOccurred())
+			computeSignSubmitConfigUpdate(network, orderer1, peer, c, "participation-trophy")
+			expectedChannelInfoPT.Height++
+
+			currentBlockNumber := nwo.CurrentConfigBlockNumber(network, peer, orderer1, "participation-trophy")
+			Expect(currentBlockNumber).To(BeNumerically(">", 1))
+
+			By("starting third orderer")
+			startOrderer(orderer3)
+			channelparticipation.List(network, orderer3, nil)
+
+			By("adding orderer3 to the consenters set")
+			channelConfig = nwo.GetConfig(network, peer, orderer2, "participation-trophy")
+			c = configtx.New(channelConfig)
+			err = c.Orderer().AddConsenter(consenterChannelConfig(network, orderer3))
+			Expect(err).NotTo(HaveOccurred())
+			computeSignSubmitConfigUpdate(network, orderer2, peer, c, "participation-trophy")
+			expectedChannelInfoPT.Height++
+
+			By("joining orderer3 to the channel as a member")
+			// make sure we can join using a config block from one of the other orderers
+			configBlockPT := nwo.GetConfigBlock(network, peer, orderer2, "participation-trophy")
+			expectedChannelInfoMember := channelparticipation.ChannelInfo{
+				Name:            "participation-trophy",
+				URL:             "/participation/v1/channels/participation-trophy",
+				Status:          "onboarding",
+				ClusterRelation: "member",
+				Height:          0,
+			}
+			channelparticipation.Join(network, orderer3, "participation-trophy", configBlockPT, expectedChannelInfoMember)
+
+			By("ensuring orderer3 completes onboarding successfully")
+			expectedChannelInfoMember.Status = "active"
+			expectedChannelInfoMember.Height = expectedChannelInfoPT.Height
+			Eventually(func() channelparticipation.ChannelInfo {
+				return channelparticipation.ListOne(network, orderer3, "participation-trophy")
+			}, network.EventuallyTimeout).Should(Equal(expectedChannelInfoMember))
+
+			// make sure orderer3 finds the leader
+			newLeader := findLeader([]*ginkgomon.Runner{ordererRunners[2]})
+			Expect(newLeader).To(Equal(leader))
+
+			By("submitting transaction to orderer3 to ensure it is active")
+			env = CreateBroadcastEnvelope(network, peer, "participation-trophy", []byte("hello-again"))
+			expectedChannelInfoPT.Height++
+			resp, err := ordererclient.Broadcast(network, orderer3, env)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(resp.Status).To(Equal(common.Status_SUCCESS))
+			expectedBlockNumPerChannel := map[string]int{"participation-trophy": int(expectedChannelInfoPT.Height - 1)}
+			assertBlockReception(expectedBlockNumPerChannel, orderers, peer, network)
```

Comment:
How about invoking ListOne again after Broadcast, to make sure the `expectedChannelInfoPT.Height` is correct?

Reply:
🧹